tensorlayer/models/vgg16.py

```diff
@@ -158,54 +173,173 @@ def conv_layers_simple_api(net_in):
             net_in.outputs = net_in.outputs - mean
 
         # conv1
-        network = Conv2d(net_in, n_filter=64, filter_size=(3, 3), strides=(1, 1), act=tf.nn.relu, padding='SAME', name='conv1_1')
-        network = Conv2d(network, n_filter=64, filter_size=(3, 3), strides=(1, 1), act=tf.nn.relu, padding='SAME', name='conv1_2')
-        network = MaxPool2d(network, filter_size=(2, 2), strides=(2, 2), padding='SAME', name='pool1')
+        net = Conv2d(net_in, n_filter=64, filter_size=(3, 3), strides=(1, 1), act=tf.nn.relu, padding='SAME', name='conv1_1')
+        if end_with in net.name:
```

Comment:
Shall we use "is" instead of "in"?

Reply:
we need to use `in`, because we are using `with tf.variable_scope` ~